cmd/ctr-remote/commands/convert.go

```diff
@@ -85,19 +102,31 @@ When '--all-platforms' is given all images in a manifest list must be available.
 			Name:  "all-platforms",
 			Usage: "exports content from all platforms",
 		},
-	},
-	Action: func(context *cli.Context) error {
+	}, samplerFlags...),
```

Comment:
I don't like this design.
`ctr images convert` should remain pure static image converter.
`ctr optimize` can have exec flags as in the current revision, but I think it should be reimplemented with `ctr image mount + ctr run + ctr images convert`.

Reply:
Sure. I'll reconsider the design of `ctr optimize` to make it containerd-based.